### DISCUSSION TITLE
Is there a way to use Thumbnail plugin with Inline?

### DISCUSSION THREAD

**Discussion by sapkauskasm** [igordanchenko/yet-another-react-lightbox#35] [Q&A] [answered]
I am struggling to get thumbnails working with Inline mode.

Thumbs are visible, but can't set their position

**Comment by igordanchenko** [comment, marked as answer]
Hi there! Is there a chance you can share a small demo reproducing the issue you are facing? And to answer your question, the Thumbnails plugin is expected to be fully compatible with the Inline plugin.

Here is a small demo - https://codesandbox.io/s/yet-another-react-lightbox-35-up041u?file=/src/App.js

**Comment by sapkauskasm** [comment]
Ok, my case is a bit more complicated.

I am trying to put  lightbox in a modal.

Here is sandbox:
https://codesandbox.io/s/basic-react-modal-example-forked-wvgjkv?file=/src/App.js

And here is what my desired result.
[![Screenshot-2022-11-01-at-10-53-54.png](https://i.postimg.cc/KjrrR8sc/Screenshot-2022-11-01-at-10-53-54.png)](https://postimg.cc/m1kFnBsK)

Maybe I am overcomplicating?

**Reply by igordanchenko** [reply]
This library comes with its own implementation of a modal. I'm just curious, is there any particular reason you want to place it inside the react-modal? Does the following demo look similar to what you are trying to build?

https://codesandbox.io/s/yet-another-react-lightbox-35-1-n85115

**Reply by sapkauskasm** [reply]
That is exactly what I am trying to build!
I thought I read documentation carefully, but missed this.

@igordanchenko 
I am checking the link, but can't figure out, what part of the code makes it to open in modal?
What ever I am trying to do, in my code - it always opens in full screen mode.

**Reply by igordanchenko** [reply]
The lightbox in the above demo also opens in its own modal. I just styled it to visually occupy 80% of the screen instead of fullscreen. You can find css styles over here - https://codesandbox.io/s/yet-another-react-lightbox-35-1-n85115?file=/src/App.css

**Reply by sapkauskasm** [reply]
After some experimenting, I got this. 
Only the class name was confusing.

Anyway, thank you for amazing library!!

**Comment by sapkauskasm** [comment]
never mind... Stupid me...
I forgot to import thumbnails CSS....
